templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -68,7 +74,15 @@ Parameters:
   KeyPairName:
     Description: Public/private key pairs allow you to securely connect to your instance
       after it launches
-    Type: AWS::EC2::KeyPair::KeyName
+    Type: String
+    Default: ''
+  BastionHostRequired:
+    Default: "true"
+    AllowedValues:
+      - "true"
+      - "false"
+    Description: Whether to provision a Bastion host instance; if 'true', then you need to provide an SSH Key Pair
+    Type: String
```

Comment:
For clarity, can we change this to:
> Whether to provision a Bastion host instance. If 'true', then you need to provide an EC2 Key Pair

Reply:
are we changing it to the `EC2 Key Pair` ?